src/main/java/frc/robot/commands/balancing/PIDSwitchBalancerCommand.java

```diff
@@ -81,4 +81,11 @@ public boolean isFinished() {
         return reachedStation && balanced;
         // return reachedStation && Math.abs(ahrs.getPitch()) <= 2.0 && stoptimer.hasElapsed(0.20);
     }
+
+    double getPower(){
```

Comment:
You don't have to manually implement this method in each subclass; inheritance will take care of it as long as it's in `BaseBalancerCommand.java`.